docs/install-manual.md

```diff
@@ -0,0 +1,144 @@
+
+# Create a swarm for development
+
+This section tells you how to create a Docker Swarm on your network to use only for debugging and testing purposes. You can also use this type of installation if you are developing custom applications for Docker Swarm or contributing to it.  
+
+> **Caution**: Only use this set up if your network environment is secured by a firewall or other measures.
+
+## Prerequisites 
+
+You install Docker Swarm on a single system which is known as your Docker Swarm manager. You create the cluster, or swarm, on one or more additional nodes on your network. Each node in your swarm must:
+
+* be accessible by the swarm manager across your network
+* the same TCP port open to listen for the swarm manager  (2375)
```

Comment:
Same here. Again, the problem was with our original docs.

`2375` was used as an example (because that's the default docker port).

You can use 1234 if you'd like on node-1 and 4444 on node-2 or whatever.

Maybe this should be a variable such as `node-port`?

`swarm join --addr=node-ip:node-port`?